### ISSUE TITLE
Received a non-Vinyl object in `dest()`

### ISSUE THREAD

**Issue by nicothin** [shinnn/gulp-gh-pages#123]
It does not work for me :(
Windows 10. node v8 and v6.

```
[23:51:58] Starting 'deploy'...
[23:52:03] [gh-pages] Cloning repo
[23:52:03] [gh-pages] Checkout remote branch `gh-pages`
[23:52:03] [gh-pages] Updating repository
[23:52:06] [gh-pages] Copying files to repository
[23:52:06] 'deploy' errored after 7.19 s
[23:52:06] Error in plugin "gulp-gh-pages"
Message:
    Received a non-Vinyl object in `dest()`
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! EpicSkills-start-kit@1.0.0 start: `gulp "deploy"`
npm ERR! Exit status 1
```

In package.json:
```
...
    "gulp": "^3.9.1",
    "gulp-gh-pages": "^0.6.0-1",
...
```

in gulpfile:
```
gulp.task('deploy', () => gulp.src('./build/**/*').pipe(ghPages()));
```

**Comment by santiagonoguez** [comment]
I also have the same problem, with similar code and configuration.

**Comment by shinnn** [comment]
Fixed in [v0.6.0-3](https://github.com/shinnn/gulp-gh-pages/releases/tag/v0.6.0-3). You'll never see the error above.
